templates/project-ts/.eslintrc.cjs

```diff
@@ -5,13 +5,19 @@ module.exports = {
     node: true,
     jest: true,
   },
-  extends: ['eslint:recommended', 'plugin:snarkyjs/recommended'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:snarkyjs/recommended',
+  ],
   parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaVersion: 'latest',
   },
   plugins: ['@typescript-eslint', 'snarkyjs'],
   rules: {
     'no-constant-condition': 'off',
+    'prefer-const': 'off',
```

Comment:
this was added by one of the `typescript-eslint` rule sets and is too opinionated IMO